server/webhook.go

```diff
@@ -101,6 +102,46 @@ func (wh *webhook) PostNotifications(p *Plugin) ([]*model.Post, int, error) {
 			continue
 		}
 
+		// Check if the user has permissions.
```

Comment:
maybe add a label to the for loop, so it becomes `continue NextNotification`?

Reply:
Something like:
![image](https://user-images.githubusercontent.com/1490756/61073706-05ff5000-a3e4-11e9-9e28-518d1ca7194a.png)

Reply:
Exactly